lib/Conversion/TritonGPUToLLVM/DotOpHelpers.h

```diff
@@ -434,13 +434,13 @@ struct MMA16816ConversionHelper {
 
   // Get a warpId for M axis.
   Value getWarpM(int M) const {
-    auto matShape = helper.getMmaMatShape();
+    auto matShape = helper.getMmaInstrShape();
```

Comment:
we probably want to call the variable `matInstrShape` now

Reply:
Done